.buildbot.sh

```diff
@@ -2,7 +2,41 @@
 
 set -eu
 
-pipx install rebench
+# Install Rust
+CARGO_HOME="$(pwd)/.cargo"
```

Comment:
I don't mind keeping this in, but the `CARGO_HOME` trick predates using Docker when we didn't want multiple runs as the same user stomping on each other. I tend to think it's unnecessary now but up to you.

Reply:
I just copied it from the `yk` CI script. We can kill it if you like. I have no preference.

Reply:
1f1c2dc90a27e3763bc5cbd24324ca0bd64d12ef should do the trick (untested).